<launch>
    <node pkg="nlink_parser" type="linktrack_aoa" name="linktrack_aoa0">
        <param name="port_name" value="/dev/ttyUSB0" />
        <param name="baud_rate" value="921600" />
    </node>
    <node pkg="nlink_parser" type="track_aoa_sub" name="track_aoa_sub" output="screen" />
</launch>